integrations/dd-sdk-android-okhttp/src/main/kotlin/com/datadog/android/okhttp/trace/TracingInterceptor.kt

```diff
@@ -387,20 +380,20 @@ internal constructor(
     }
 
     private fun extractParentContext(tracer: Tracer, request: Request): SpanContext? {
-        val tagContext = request.tag(Span::class.java)?.context()
-            ?: request.tag(TraceContext::class.java)?.toOpenTracingContext()
+        val tagContext = request.tag(Span::class.java)?.context() ?: request.tag(TraceContext::class.java)
```

Comment:
why do we still check for `request.tag(Span::class.java)` ? this was from OpenTracing if I am not mistaken and should not be here anymore.

Reply:
this is just a typealias magic. Removed all of that confusing lines